### PR TITLE
update SORManager to use new SOR

### PR DESCRIPTION
**Pull request by TomAFrench**
# Description

This draft PR is to allow us to ensure that as we make changes to the SOR interface it doesn't become difficult to reconcile those with the frontend.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other

## How should this be tested?

Check that prices quoted on trading page match that of the production app.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] My changes generate no new console warnings
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/composables/trade/useSor.ts [hunk 1]
```diff
@@ -169,9 +168,10 @@ export default function useSor({
     const poolsUrlV1 = `${
       configService.network.poolsUrlV1
     }?timestamp=${Date.now()}`;
-    const poolsUrlV2 = `${
-      configService.network.poolsUrlV2
-    }?timestamp=${Date.now()}`;
+    // TODO: pull a set of SubgraphPoolBase[] data from here
+    // const poolsUrlV2 = `${
```
Comment (TomAFrench) [inline src/composables/trade/useSor.ts:172]: This didn't seem to get used before (as we'd always overwrite with subgraph data). Shall we just get rid of these?

File: package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
   "dependencies": {
     "@balancer-labs/assets": "github:balancer-labs/assets#master",
     "@balancer-labs/sor": "^1.1.0",
-    "@balancer-labs/sor2": "github:balancer-labs/balancer-sor#john/ui-694-sor-lido-add-static-route-refactor",
+    "@balancer-labs/sor2": "github:balancer-labs/balancer-sor#scratch-branch",
```
Comment (TomAFrench) [inline package.json:25]: Ideally we could push this to npm before merging.

**Comment by TomAFrench** [comment]
Looks like some things got mucked up in the rebase.

File: src/composables/trade/useSor.ts (hunk 1 above)
Comment (johngrantuk) [reply src/composables/trade/useSor.ts:172]: Yeah, I think this was leftover from a previous method of getting the data.

Comment (TomAFrench) [reply src/composables/trade/useSor.ts:172]: Cool, I'll get rid of the V2 version

**Review by garethfuller — APPROVED** [review]

**Pull request merged**